build/common/scripts/lint_copyright_banner.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Copyright Contributors to the Open Cluster Management project
+
+
+set -e
+
+ec=0
+for fn in "$@"; do
+  if ! grep -L -q -e "Apache License, Version 2" "${fn}"; then
+    echo "Missing license: ${fn}"
+    ec=1
+  fi
+
+  if ! grep -L -q -e "Copyright" "${fn}"; then
+    echo "Missing copyright: ${fn}"
+    ec=1
+  fi
+done
+
+exit $ec
```

Comment:
It checks that any of the following copyrights are in the header of source files. I think this may be required for it be open source.
```
lic_ibm_identifier=" (c) Copyright IBM Corporation"
lic_redhat_identifier_2020=" Copyright (c) 2020 Red Hat, Inc."
lic_redhat_identifier_2021=" Copyright (c) ${current_year} Red Hat, Inc."
```

Reply:
That's the other copyright file 🙂

Reply:
I deleted my previous comment because I thought were talking about a different file.

So this is useful since it checks to see that Apache license header is present which I think is required by Red Hat for open source projects. I think the same is true for the Copyright header.

Reply:
Okay--I'm not sure. There have been lots of back-and-forth about copyrights in our move to being open, and lots of Red Hat projects I see don't even have one (Hive, for example). I think we're covered here by the LICENSE file in the repo. But you're right--it'd probably be good to check with someone.